test/test_browser.py

```diff
@@ -5406,11 +5406,15 @@ def test_wasmfs_fetch_backend(self, args):
                     args=['-sWASMFS', '-pthread', '-sPROXY_TO_PTHREAD', '-sINITIAL_MEMORY=32MB',
                           '--js-library', test_file('wasmfs/wasmfs_fetch.js')] + args)
 
-  @requires_threads
   @no_firefox('no OPFS support yet')
-  def test_wasmfs_opfs(self):
+  @parameterized({
+    '': (['-pthread', '-sPROXY_TO_PTHREAD'],),
+    'jspi': (['-Wno-experimental', '-sASYNCIFY=2'],),
+  })
+  @requires_threads
```

Comment:
Won't this add threads even in the jspi version?

Reply:
`@requires_threads` only checks for the environment variable EMTEST_LACKS_THREAD_SUPPORT. `@also_with_threads` is the one that does require and adds pthreads.